.github/workflows/foundation-ang-trigger.yml

```diff
@@ -0,0 +1,53 @@
+# Controls when the action will run. The workflow runs when manually triggered using the UI or API
+
+name: Foundation_Ang_On_Demand
+
+on:
+  push:
+    branches:
+      - dev/angular13
+      - dev/angular12
```

Comment:
Shouldn't those branches be part of the PR as well?

Reply:
Yes, They are already in place and sync with the Angular versions.

Reply:
I am a bit lost here... So they were already created in some other PR?
Is there any other reason to have them apart from testing different versions?

Reply:
There are two points: 

1. The reason for using branches (not tags) is by default they are available as one of the options to use the workflow. Next to that, we also have the foundation-ang version. So, the testing would be straightforward to use the pair of branches and the foundation-ang version. 
![image](https://user-images.githubusercontent.com/6618162/195342929-1fa17f41-2f9b-4892-b919-1999dcf81be6.png)

2. I created those two branches manually. I agree with you that we can create them automated (based on git tag latest commits that use those angular versions) but we should take into account that every six months we need to have a new branch.